apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -6527,10 +6527,10 @@
         }
```

Comment:
you can remove 
```
"options": {
        "deleteBackup": {
          "title": "Delete your backup",
          "description": "All synced accounts will be removed. This won’t affect your local accounts."
        }
      }
```

If I'm not wrong :)

Reply:
Oh yes missed that

Reply:
✅